sdk-installer/setup-git-sdk.bat

```diff
@@ -5,13 +5,13 @@
 @REM https://technet.microsoft.com/en-us/library/bb490909.aspx says:
 @REM <percent>~dpI Expands <percent>I to a drive letter and path only.
 @REM <percent>~fI Expands <percent>I to a fully qualified path name.
-@FOR /F "delims=" %%D in ("%~dp0") do @set cwd=%%~fD
+@FOR /F "delims=" %%D in ("%~dp0") do @set "cwd=%%~fD"
```

Comment:
Sorry for missing this PR!

So I guess the underlying issue is that the script does not run correctly e.g. when installing into a path whose name has spaces in it?

I am not sure if both this change and the quoting of the `PATH` assignment are needed... When I run

```cmd
set PATH=%PATH%;C:\with spaces
```

it works over here...

Reply:
Running `et PATH=%PATH%;C:\with spaces` will set the path to `et PATH=%PATH%;C:\with`